docs/pages/includes/discovery/discovery-config.yaml

```diff
@@ -75,3 +75,16 @@ discovery_service:
         # GCP resource tag filters used to match resources.
         tags:
           "*": "*"
+    # Matchers for discovering services inside Kubernetes cluster and exposing them as Teleport apps
+    # When `kubernetes` value is set `discovery_group` parameter is mandatory, it should be set to 
+    # the name of Kubernetes cluster discovery service is running in.
+    kubernetes:
+      # Type of services to discover. Currently, only "app" is supported. Default value is `["app"]`
+    - types: ["app"]
+      # List of namespaces of the Kubernetes cluster to search in. Default value is `*` to search all namespaces.
+      namespaces: ["test", "staging"]
+      # List of Kubernetes labels based on which to filter out services. Labels will be copied to Teleport app 
```

Comment:
So the Discovery Service queries Kubernetes for services with the labels in this list, and creates application resources with the corresponding Teleport labels? I would be more explicit about this in the comment.

Reply:
I changed the description and removed mentioning of copying labels to apps (it can be confusing, because all service labels will be copied to an app, not just listed in the matcher, and I don't think it's even a right place for this information, here we only care about finding targeted services). I hope it looks better now.